client/src/components/contrib/contrib.js

```diff
@@ -2,6 +2,7 @@ import React, { useState, useEffect } from 'react';
 import ContribCard from './contrib-card.js';
 import FadeIn from 'react-fade-in';
 
+import {useWindowScroll} from "react-use";
```

Comment:
Where are you using this `useWindowScroll` here?

Reply:
I tried to make another thing and sorry I forgot to remove it but I didn't use it in this issue @fave77